src/Player.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Player.hpp"
+
+Player::Player(const std::string &color)
+        : color_(color),
+        money_(3000),
+        location_(0)
+{}
+
+std::string Player:: getColor() const {return color_;}
+int Player:: amountOfMoney() const {return money_;}
+int Player :: getLocation() const { return location_;}
+bool Player :: ownsProperty() const
+{
+    if(properties_.empty()) return false;
+    else return true;
+}
+
+void Player::setLocation(const int &location)
```

Comment:
Literal types like int, double, float or char does not need to be passed as a reference. Copy should be as fast as a reference and is advised in this case.